src/types/world.ts

```diff
@@ -2,6 +2,10 @@ import { EntranceStepConfig } from "types/EntranceStep";
 import { Question } from "types/Question";
 import { UserStatus } from "types/User";
 
+import { Brand } from "utils/types";
+
+export type WorldSlug = Brand<string, "WorldSlug">;
```

Comment:
How about moving these types to the `branding.ts` instead for a more centralized overview of things that are same but not same?

```suggestion
export type WorldSlug = WithBrand<string, "WorldSlug">;
```

Reply:
They're part of the greater `World` type in my opinion. If `World` did not exist, these wouldn't either.